Common/src/CMultiGridQueue.cpp

```diff
@@ -164,7 +166,8 @@ bool CMultiGridQueue::EmptyQueue(void) const {
 
   if (QueueCV.size() == 1) {
     for (auto iPoint : QueueCV[0])
-      if (RightCV[iPoint]) return false;
+      if ((iPoint != QueueType::ErasedValue) && RightCV[iPoint])
+        return false;
```

Comment:
This is not related to this PR, it's a fix for a problem that @economon found.

Reply:
👍